contribs/gmf/src/services/treemanager.js

```diff
@@ -114,10 +114,12 @@ gmf.TreeManager.prototype.addTheme = function(theme) {
  * already in the tree.
  * @param{Array.<GmfThemesNode>} groups An array of gmf theme nodes.
  * @param{boolean=} opt_add if true, force to use the 'add' mode this time.
+ * @param{boolean=} showNotification if false notifyCantAddGroups_ is not called
  * @export
  */
-gmf.TreeManager.prototype.addGroups = function(groups, opt_add) {
+gmf.TreeManager.prototype.addGroups = function(groups, opt_add, showNotification) {
   var groupNotAdded = [];
+  showNotification = typeof showNotification !== 'undefined' ? showNotification : true;
```

Comment:
This could be:

     showNotification = showNotification !== 'undefined' ? showNotification : true;